src/System.Reflection/tests/FieldInfo/FieldInfo_MethodTests.cs

```diff
@@ -305,26 +184,22 @@ private static FieldInfo GetField(string field)
         private int _privateInt = 1; // Field for Reflection
         private string _privateStr = "privateStr"; // Field for Reflection
 
-        private static Object s_field_Assembly1 = null;				    // without keyword
-        private static Object s_field_Assembly2 = null;			// with private keyword
-        protected static Object Field_Assembly3 = null;			// with protected keyword
-        public static Object Field_Assembly4 = null;			// with public keyword
-        internal static Object Field_Assembly5 = null;			// with internal keyword
-
-        private static Object s_field_FamilyAndAssembly1 = null;						// without keyword
-        private static Object s_field_FamilyAndAssembly2 = null;			    // with private keyword
-        protected static Object Field_FamilyAndAssembly3 = null;			// with protected keyword
-        public static Object Field_FamilyAndAssembly4 = null;				// with public keyword
-        internal static Object Field_FamilyAndAssembly5 = null;				// with internal keyword
-
-        private static Object s_field_FamilyOrAssembly1 = null;				    // without keyword
-        private static Object s_field_FamilyOrAssembly2 = null;			// with private keyword
-        protected static Object Field_FamilyOrAssembly3 = null;			// with protected keyword
-        public static Object Field_FamilyOrAssembly4 = null;			// with public keyword
-        internal static Object Field_FamilyOrAssembly5 = null;			// with internal keyword
-
-        public readonly int rointField = 1;
-
-        public const int constIntField = 1222;
+        private static object s_field_Assembly1 = null;				    // without keyword
+        private static object s_field_Assembly2 = null;			// with private keyword
+        protected static object Field_Assembly3 = null;			// with protected keyword
+        public static object Field_Assembly4 = null;			// with public keyword
+        internal static object Field_Assembly5 = null;			// with internal keyword
+
+        private static object s_field_FamilyAndAssembly1 = null;						// without keyword
+        private static object s_field_FamilyAndAssembly2 = null;			    // with private keyword
+        protected static object Field_FamilyAndAssembly3 = null;			// with protected keyword
+        public static object Field_FamilyAndAssembly4 = null;				// with public keyword
+        internal static object Field_FamilyAndAssembly5 = null;				// with internal keyword
+
+        private static object s_field_FamilyOrAssembly1 = null;				    // without keyword
+        private static object s_field_FamilyOrAssembly2 = null;			// with private keyword
+        protected static object Field_FamilyOrAssembly3 = null;			// with protected keyword
+        public static object Field_FamilyOrAssembly4 = null;			// with public keyword
+        internal static object Field_FamilyOrAssembly5 = null;			// with internal keyword
```

Comment:
Turns out these variables aren't actually used for the tests so I just took them out, but I updated their names in `FieldInfo_PropertyTests`